tpl-implementations/include/experimental/__p1673_bits/kokkos-kernels/blas2_matrix_rank_1_update.hpp

```diff
@@ -0,0 +1,174 @@
+#ifndef LINALG_TPLIMPLEMENTATIONS_INCLUDE_EXPERIMENTAL___P1673_BITS_KOKKOSKERNELS_BLAS2_MATRIX_RANK_1_UPDATE_HPP_
```

Comment:
Would you consider adding a copyright header to the top of new files?

Reply:
Fixed on https://github.com/kokkos/stdBLAS/pull/157/commits/4142d9a1481873a53bfb7d5958016c5f1a89fa60